README.md

```diff
@@ -378,7 +378,7 @@ Flutter ile ilgili Türkçe makaleler kategorilerine göre ayrılmış halde bur
 - [ECharts](https://github.com/entronad/flutter_echarts) - Zengin bir içeriğe sahip grafik kütüphanesi.
 
 #### Progress
-- [Progress Dialog](https://github.com/fayaz07/progress_dialog)
+- [Progress Dialog](https://github.com/emreesen27/Flutter-Progress-Dialog)
```

Comment:
We can merge it but the naming of your project seems like a problem for Flutter projects :)

Reply:
Are you talking about the name of the repository? It is published on pub.dev in accordance with the standards.
[sn_progress_dialog](https://pub.dev/packages/sn_progress_dialog)